### PR TITLE
regular review of reward guideline

### PR DESCRIPTION
**Pull request by darwintree**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://doc.confluxnetwork.org/docs/general/CONTRIBUTING#create-a-pull-request).
- [x] A relating issue is created: #507 (<- fill the issue number here)
- [x] **This is NOT a PR to submit translation**

- - -
This change is [Reviewable](https://reviewable.io/reviews/Conflux-Chain/conflux-documentation/508)

### CODE REVIEW

**Comment by darwintree** [comment]
This PR primarily aims to address the issues mentioned in #507:

1. Detailing how rewards for website improvements will be distributed has been added, applicable to both issues and pull requests.
2. The merging of duplicate entries for errors and improvements for both issues and pull requests.
3. We've removed the general statement of "doubling rewards for issues with solutions". For basic issues or errors, the solution is often evident within the issue itself. This statement is now attached to the necessary entries.
4. We've clarified the methodology used to delineate rewards, which includes:
    1. Adding examples showing how rewards will be determined based on the most recent guidelines.
    2. Providing clearer statements regarding the guidelines.
    3. Addressing variations in reward amounts, typically featuring an increased base value and decreased maximum value to provide contributors with a more stable expectation. An exception to this is the maximum value for new articles, which has been raised to encourage the creation of high-quality tutorials.
5. The change for reward distribution process. Now Dework is deprecated and we'll track issues using notion tables. The maximum interval for reward distribution process is now 3 months but we'll try to shorten it.

**Comment by darwintree** [comment]
@jackleeio 
As now you are the primary external contributor for our repository, we would know your opinion on the new version of the reward guideline. Please let us know if any entry is not reasonable or doesn't match the contributors' effort

**Comment by jackleeio** [comment]
it’s much clearer than the previous version

File: docs/general/CONTRIBUTING/reward-guidelines.md [hunk 1]
```diff
@@ -13,70 +13,96 @@ This article details how your contributions, categorized under 'Issues' and 'Pul
...
+  * Suggesting improve the document structure: 32 FC for *valuable*, but not necessarily ACCEPTED solutions, or 64 FC if accepted. NOTE: We will carefully consider these issues.
+  * Highlighting current governance shortcomings: Determined case by case.
+* **Document Website Code Issues**
+  * Front-end style issues including layout, font size, line spacing, etc: 48 FC, or 96 FC with a suggested solution (not necessarily a pull request)
+  * Recommending specific components for certain functions: 96 FC.
+  * Other issues: Determined case by case.
+
+### Pull Requests Reward Guidelines
+
+Reward values for pull requests are primarily determined by the complexity and significance of the content. Please refer to [rewards evaluation](#rewards-evaluation-and-examples) for more details on value determination..
```
Comment (jackleeio) [inline docs/general/CONTRIBUTING/reward-guidelines.md:46]: > Please refer to rewards evaluation for more details on value determination..

There's an extra period

**Pull request merged**